### PR TITLE
#11: add location permission request

### PR DESCRIPTION
**Pull request by frayeralex**

### CODE REVIEW

**Comment by frayeralex** [comment]
[User location permission request handle](https://trello.com/c/TVWV6MTN/11-user-location-permission-request-handle)

**Review by SuprunOleh — COMMENTED** [review]
hope it works?

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt [hunk 1]
```diff
@@ -79,6 +83,51 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
                 Log.w(TAG, "Failed to read value.", error.toException())
             }
         })
+
+        checkLocationPermission()
+    }
+
+    private fun checkLocationPermission() {
```
Comment (SuprunOleh) [inline app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt:90]: This is looking good

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt [hunk 2]
```diff
@@ -79,6 +83,51 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback {
...
+                    // todo show marker
+                } else {
+                    // todo
+                    // permission denied, boo! Disable the
+                    // functionality that depends on this permission.
+                }
+                return
+            }
+            else -> { }
+        }
```
Comment (SuprunOleh) [inline app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt:130]: ok

**Review by SuprunOleh — APPROVED** [review]
одобрямс

**Pull request merged**